components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/sourceview_api.jsp

```diff
@@ -121,6 +121,18 @@
         var isValidXML = isValidXml(trim(source));
         if (!isValidXML) {
             return false;
+        } else {
```

Comment:
This code does not work a=on all browsers it seems.
http://stackoverflow.com/questions/7022568/activexobject-in-firefox-or-chrome-not-ie